docs/entity_registry_index.md

```diff
@@ -20,3 +20,11 @@ Good sources for a unique ID:
 - latitude/longitude
 
 If a device has a single serial but provides multiple entities, combine the serial with unique identifiers for the entities. For example, if a device measures both temperature and humidity, you can uniquely identify the entities using `{serial}-{sensor_type}`.
+
+```python
+# Example, inside a component
+    @property
+    def unique_id(self):
+        """Return the ID of this entity."""
+        return self._unique_idty
+```
```

Comment:
This seems a little disconnected from the example outlined in the previous line. Perhaps either move up to line 7 (adding an explanation in line 6) or even better expand this example to show prefixing an entity with the serial

Reply:
@TomBrien - Now that Paulus has added a link to the other docs that show `entity_id` explicitly, we can remove this example and reduce the maintenance burden. Can you propose a change that takes this away so I can simply commit/accept it please? Then we're done, I believe. Thanks.

Reply:
Agreed :)

Reply:
@TomBrien - I might not have been clear... I would like you to make the edit like you did for another one so I can click "Commit suggestion" to get rid of this. Is that OK?

Reply:
Sorry my bad here you go
```suggestion
```